io.qt.qtwebengine.BaseApp.json

```diff
@@ -0,0 +1,87 @@
+{
+    "id" : "io.qt.qtwebengine.BaseApp",
+    "branch": "5.13",
+    "runtime" : "org.kde.Platform",
+    "sdk" : "org.kde.Sdk",
+    "runtime-version" : "5.13",
+    "modules" : [
+        {
+            "name" : "qt5-qtwebengine",
+            "buildsystem" : "qmake",
+            "cleanup-platform" : [
+                "/bin",
+                "/mkspecs"
+            ],
+            "make-install-args": ["INSTALL_ROOT=/app/potatoes"],
```

Comment:
Uh, is it really what you want?

Reply:
Define "want"

Reply:
I guess the question is if you really want to install into dir called `potatoes` and why.

Reply:
I guess the question is if you really want to install into dir called `potatoes` and why.

Reply:
It insists to install to /usr, it's hardcoded in Qt and I don't see a way around it. We need to install elsewhere and then move it into /app and place it properly. Hence using this name. I can change it if it bothers you.

Reply:
Yeah, calling it `scratchdir` would be better. BaseApps are often used as templates later.